packages/appium/docs/en/intro/index.md

```diff
@@ -2,4 +2,200 @@
 title: Intro to Appium
 ---
 
-TODO
+As mentioned on the main page, Appium is an open-source project and ecosystem of related software,
+designed to facilitate UI automation of many app platforms. With the release of Appium 2.0, Appium
+has the following primary goals:[^1]
+
+- Make platform-specific automation capabilities available under a cross-platform, standard API
+- Allow easy access to this API from any programming language
+- Provide tools to enable convenient community development of Appium extensions
+
+[^1]:
+    To meet these primary goals, we also work with a set of secondary goals or methodology
+    principles, which we also encourage for Appium extension developers:
+
+    - As far as possible, rely on (and contribute to) open source technology
+    - As far as possible, rely on vendor-provided tools for a given platform
+    - As far as possible, avoid modification to the app for the purpose of automation
```

Comment:
I don't understand this

Reply:
it just means try not to force the user to build something into their app or recompile it so that it is testable. used to be a lot more important but at the end of the day we have to do what we have to do for a given platform. it probably doesn't need to be said anymore.